src/Transports.AspNetCore.NewtonsoftJson/GraphQLRequestDeserializer.cs

```diff
@@ -18,7 +18,15 @@ public class GraphQLRequestDeserializer : IGraphQLRequestDeserializer
 
         public GraphQLRequestDeserializer(Action<JsonSerializerSettings> configure)
         {
-            var settings = new JsonSerializerSettings();
+            var settings = new JsonSerializerSettings()
+            {
+                DateFormatHandling = DateFormatHandling.IsoDateFormat,
+                DateParseHandling = DateParseHandling.None,
```

Comment:
👍 ping @slavautesinov

Reply:
Yeah, it's another mentioned possible solution